clients/venice-push-job/src/main/java/com/linkedin/venice/hadoop/VenicePushJob.java

```diff
@@ -3421,10 +3417,9 @@ private void killJobAndCleanup(
         LOGGER.info("Offline push job has been killed, topic: {}", topicInfo.topic);
       }
     }
-    close();
```

Comment:
This is the crux of the fix

Reply:
We should still be calling close() after sending report, right?

Reply:
Calling `close` is the responsibility of the code that creates the `VenicePushJob` object. It seems weird that we would close an object during it's execution